src/presentational/metric/MetricsList.css

```diff
@@ -0,0 +1,20 @@
+.metrics-list-container{
+    grid-template-columns: 1fr;  
+    width: 100%;
+    display: grid;
+    grid-gap: 20px;
+}
+
+@media(min-width: 500px){
+.metrics-list-container {
+    /*auto divide the space to 2 columns each is min 110px*/
+    /* https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_Layout */
+    grid-template-columns: minmax(110px, 1fr) minmax(110px, 1fr);
+ }
+}
+
+@media(min-width: 800px){ 
+    .metrics-list-container{
+        grid-template-columns: 1fr 1fr 1fr 1fr;  
```

Comment:
if big screen we display the boxes horizontally
if the screen is smaller we display them in 2 columns
other wise in small mobile screens its 1 per row